layouts/partials/components/carousel.html

```diff
@@ -0,0 +1,42 @@
+<section>
+	<div id="tinyslider-carousel">
+		{{ range sort $.Params.carousel.images "weight" }}
+		<div class="item">
+			<div>
+				<img class="img-responsive" src="{{ .image }}" alt="{{ .description }}" />
+			</div>
+			{{ if .description }}
+			<p>{{ .description }}</p>
+			{{ end }}
+		</div>
+		{{ end }}
+	</div>
+	<div class="tinyslider-controls">
+		<ul class="thumbnails" id="customize-thumbnails">
+		{{ range sort $.Params.carousel.images "weight" }}
+			<li class="item"><img class="img-responsive" src="{{ .image }}" alt="{{ .description }}" width="50px" /></li>
+		{{ end }}
+		</ul>
+		<ul class="controls" id="customize-controls">
+			{{- $angleleft := resources.Get "images/angle-left.png" }}
+			{{- $angleright := resources.Get "images/angle-right.png" }}
+			<li class="prev"><img src="{{ $angleleft.Permalink }}" alt="Move selection to left" /></li>
+			<li class="next"><img src="{{ $angleright.Permalink }}" alt="Move selection to right" /></li>
```

Comment:
I can not comment on an image so I comment here.

Do you think it is possible to use an icon instead of an image here? It will not only increase the repo size (Hugo has limitations on theme repo size) but also increase the time users wait for the page to load.

If using an icon is possible, font awesome may be a good choice because we already use icons from font awesome so users will not need to wait for another package to load. For example, [angle-right](https://fontawesome.com/icons/angle-right?style=solid) may be a good option. You just need change it to `<i class="fas fa-angle-right"></i>` to use it.

Reply:
Great, did not think that fonts have arrows defined. So changed in c89b1fd